### ISSUE TITLE
Allow theme to control style and formatting of image alt text

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#730]
Allow the theme to control the style and formatting of image alt text. In particular, it should be possible to control the color and the string template.

```
image:
  alt:
    color: #ff0000
    content: '[{alt}] <em>{target}</em>'
```

We have to think about what the right reference syntax for placeholders would be.
